index.html

```diff
@@ -149,6 +147,22 @@ <h3 class="skills_list">Skills<p><i class="arrow right"></i></p>
             </h3>
         </div>
     </section>
+    <footer class="contact-form">
+        <section class="Contact_form">
+            <h1 class="contact_me">Contact me</h1>
+            <p class="contact">If you have an application you are interested in developing, a feature that you need
+                built or a project that needs coding. I’d love to help with it.</p>
+            <form action="https://formspree.io/f/xeqdkkpj" method="POST">
+                <ul class="form_fields">
+                    <li><input class="name" name="for-name" type="text" placeholder="Your Name" maxlength="30" required>
+                    </li>
+                    <li><input class="email" name="for-email" type="email" placeholder="Your Email" required></li>
+                    <li><textarea class="message" name="for-msg" id="Msg" cols="30" rows="8"
+                            placeholder="Write your message here" maxlength="500" required></textarea></li>
+                </ul>
+                <button class="button" id="btn" type="button">Get In Touch</button>
```

Comment:
- [ ] Kindly make sure the role of your button is to submit the filled information. While the type of your button is `button` we can't be able to submit the form. I highly recommend changing the button type to `submit` after that you will be able to submit your form correctly. :wink:

Reply:
fixed